portal/v1/src/select/select.js

```diff
@@ -0,0 +1,46 @@
+const select_input = $("#enterResourceId");
+const select_element = $("#selResourceId");
+
+select_element.css("top", select_input.outerHeight());
+
+function selectResourceId() {
+    select_input.val(select_element.find(":selected").text());
+    select_element.css("visibility", "hidden");
+}
+
+select_input.on("keyup change", function () {
+    if (select_input.val().length >= 2) {
+        select_element.children().each(function (i) {
+            if (i != 0) {
+                if (!$(this).text().toLowerCase().match(select_input.val().toLowerCase())) {
```

Comment:
nit: can we invert this condition? imo it is better to avoid using leading `!` as they can be sneaky. 

e.g. `if (textIsEqual) { show } else { hide }`